src/client/components/ContactForm/ContactForm.styles.css

```diff
@@ -0,0 +1,47 @@
+form {
```

Comment:
You could use flexbox instead of grid, you can learn more about using flexbox `[here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)`. Personally, I think it helps to make the code easier to mantain.

Reply:
Sure- I will keep it mind from next time